conf.yaml

```diff
@@ -818,6 +818,18 @@ contents:
               -
                 repo:   ecctl
                 path:   docs/
+          - title:      Elastic Cloud Terraform Provider
+            prefix:     en/tpec
+            tags:       CloudTerraform/Reference
+            subject:    TPEC
+            current:    master
+            branches:   [ master ]
+            index:      docs/index.asciidoc
+            chunk:      1
+            sources:
+              -
+                repo:   terraform-provider-ec
+                path:   docs/
```

Comment:
Depends on https://github.com/elastic/terraform-provider-ec/pull/163#pullrequestreview-522134387

Reply:
Yup, I updated, content now lives in https://github.com/elastic/terraform-provider-ec/pull/163/files#diff-549ff6e8034edcde5d1dbd30d3546302ffccfad46d8f0d0845133f39e4521cd5.